sigopt_sklearn/search.py

```diff
@@ -257,7 +273,8 @@ def _fit(self, X, y, parameter_iterable=None):
         pre_dispatch = self.pre_dispatch
 
         # setup SigOpt experiment and run optimization
-        conn = sigopt.Connection(client_token=self.client_token)
+        conn = (self.sigopt_connection if self.sigopt_connection
```

Comment:
Should be addressed now by https://github.com/sigopt/sigopt_sklearn/pull/3/commits/16c76585cf793ecf3c5bb0bbcbad35dbaea526d0.